src/test/scala/org/kframework/minikore/TextToMiniTest.scala

```diff
@@ -70,7 +70,7 @@ class TextToMiniTest {
             |ERROR: Line 4: Column 8: Expected 't', but ' '
             |  impor t B []
             |       ^
-            |""".stripMargin.stripPrefix("\n").stripSuffix("\n"),
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
```

Comment:
Ok, I can go with the current fix, since it will work mostly unless one has weird text editor settings.